portal/src/texts/core/clarity.md

```diff
@@ -1,5 +1,19 @@
 ## Klarhet
 
-Et av Jøkuls viktigste prinsipper er klarhet. Det skaper vi blant annet med luft. Luft og mellomrom er viktig i design, det bryter opp og gir fokus på bestemte elementer.
+Fremtind skal fremstå som ærlig og transparent med en klar og varm stemme. Vi skal skape klarhet gjennom forenkling – vi fjerner unødvendig støy og gir brukeren kun relevante valg.
 
-Vi vektlegger klarhet også med språket vi bruker. Klarspråk er hovedprinsippet for Fremtinds stil og tone, som du finner i Stemmen vår
+Bruk av rikelig med luft ("Whitespace") bidrar til å dele opp og porsjonere innhold som skaper en behagelig leserytme. Gjennom gode og tydelige designløsninger gir vi brukeren oversikt og en følelse av trygghet.
+
+### Nøkkelord
+
+-   Forenkling
+-   Styre fokus
+-   Transparent
+
+### I bruk
+
+I praksis betyr dette prisnippet at vi ikke skal overlesse brukeren med informasjon. Ha få elementer på skjermen til enhver tid, og sørg for at størrelse, posisjon og avstander skaper god oversikt og rytme.
+
+Som designer kan du benytte deg av skalaene for [typografi]() og [sperring]() for å sørge for konsistente og konsekvente avstander og størrelsesforhold.
```

Comment:
Konsistente og konsekvente blir det samme her - det holder med konsekvente. Eventuelt velg et annet ord for konsistent hvis du vil ha to ord der. Synonymer til konsistent: Konket og ensartet. Du kan også velge tydelig, eller samsvarende.

Reply:
Velger konsekvent i så fall. Men slik jeg forsto det betyr konsekvent "logisk gjennomtenkt" og konsistent "ensartet/sammenhengende"? Var i hvert fall derfor jeg tok med begge to 😅